CMakeLists.txt

```diff
@@ -123,25 +123,28 @@ if( VIAME_ENABLE_PYTHON )
 
   option( VIAME_SYMLINK_PYTHON "Symlink python files instead of copying." OFF )
   mark_as_advanced( VIAME_SYMLINK_PYTHON )
+  # set flag so we can use kwiver python utils
+  set(KWIVER_SYMLINK_PYTHON VIAME_SYMLINK_PYTHON)
```

Comment:
This variable is set but not used?

Reply:
Further clarrification, it's not used at the super-build level, but must be set down in the VIAME subproject

Reply:
KWIVER_SYMLINK_PYTHON should be set if you call kwiver_add_python_module, so it is used in the VIAME subproject. Did I put it in the wrong place?

Reply:
Yeah it needed to be in the VIAME subproject not at the super-build level (inside the large else statement lower down)